docs/operators-and-hooks-ref.rst

```diff
@@ -530,63 +530,63 @@ These integrations allow you to copy data from/to Amazon Web Services.
    * - `Amazon DynamoDB <https://aws.amazon.com/dynamodb/>`__
      - `Amazon Simple Storage Service (S3) <https://aws.amazon.com/s3/>`_
      -
-     - :mod:`airflow.providers.amazon.aws.operators.dynamodb_to_s3`
+     - :mod:`airflow.providers.amazon.aws.transfers.dynamodb_to_s3`
 
    * - `Amazon Redshift <https://aws.amazon.com/redshift/>`__
      - `Amazon Simple Storage Service (S3) <https://aws.amazon.com/s3/>`_
      -
-     - :mod:`airflow.providers.amazon.aws.operators.redshift_to_s3`
+     - :mod:`airflow.providers.amazon.aws.transfers.redshift_to_s3`
 
    * - `Amazon Simple Storage Service (S3) <https://aws.amazon.com/s3/>`_
      - `Amazon Redshift <https://aws.amazon.com/redshift/>`__
      - :doc:`How to use <howto/operator/amazon/aws/s3_to_redshift>`
-     - :mod:`airflow.providers.amazon.aws.operators.s3_to_redshift`
+     - :mod:`airflow.providers.amazon.aws.transfers.s3_to_redshift`
 
    * - `Amazon Simple Storage Service (S3) <https://aws.amazon.com/s3/>`_
      - `Snowflake <https://snowflake.com/>`__
      -
-     - :mod:`airflow.providers.snowflake.operators.s3_to_snowflake`
+     - :mod:`airflow.providers.snowflake.transfers.s3_to_snowflake`
 
    * - `Amazon Simple Storage Service (S3) <https://aws.amazon.com/s3/>`_
      - `Apache Hive <https://hive.apache.org/>`__
      -
-     - :mod:`airflow.providers.apache.hive.operators.s3_to_hive`
+     - :mod:`airflow.providers.apache.hive.transfers.s3_to_hive`
 
    * - `Amazon Simple Storage Service (S3) <https://aws.amazon.com/s3/>`__
      - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
      - :doc:`How to use <howto/operator/gcp/cloud_storage_transfer_service>`
-     - :mod:`airflow.providers.google.cloud.operators.s3_to_gcs`,
-       :mod:`airflow.providers.google.cloud.operators.cloud_storage_transfer_service`
+     - :mod:`airflow.providers.google.cloud.transfers.s3_to_gcs`,
+       :mod:`airflow.providers.google.cloud.transfers.cloud_storage_transfer_service`
```

Comment:
Cna you check it?

Reply:
What do you mean by OK ? It looks good. What do you suspect is wrong here?

Reply:
Cloud transfer service operators should be in operators package.